### PR TITLE
Waveguide: Add convenience methods for 90° left and right turns

### PR DESCRIPTION
**Pull request by fbeutel**
I frequently need to perform left and right turns  and I think these methods would improve readability of the code

### CODE REVIEW

File: gdshelpers/parts/waveguide.py [hunk 1]
```diff
@@ -546,6 +546,18 @@ def add_straight_segment_until_level_of_port(self, port, **line_kw):
         self.add_straight_segment_to_intersection(port.origin, port.angle - np.pi / 2, **line_kw)
         return self
 
+    def add_left(self, bend_radius):
+        """
+        Add a 90° left turn with the given bend radius
+        """
+        return self.add_bend(0.5*np.pi, bend_radius)
```
Comment (frank-bp) [inline gdshelpers/parts/waveguide.py:553]: `
```suggestion
    def add_left_bend(self, radius, angle=np.pi/2):
        """
        Add a left turn (90° or as defined by angle) with the given bend radius
        """
        return self.add_bend(angle, radius)
```

File: gdshelpers/parts/waveguide.py [hunk 2]
```diff
@@ -546,6 +546,18 @@ def add_straight_segment_until_level_of_port(self, port, **line_kw):
...
+
+    def add_right(self, bend_radius):
+        """
+        Add a 90° right turn with the given bend radius
+        """
+        return self.add_bend(-0.5*np.pi, bend_radius)
```
Comment (frank-bp) [inline gdshelpers/parts/waveguide.py:559]:
```suggestion

    def add_right_bend(self, radius, angle=np.pi/2):
        """
        Add a right turn (90° or as defined by angle) with the given bend radius
        """
        return self.add_bend(-angle, radius)
```

**Review by frank-bp — CHANGES_REQUESTED** [review]
Nice idea, but a "flexible" angle would be nice for directional couplers and similar structures.

**Comment by HelgeGehring** [comment]
Looks good, thanks!

**Pull request merged**